data_steward/common.py

```diff
@@ -25,7 +25,7 @@
 VOCABULARY_TABLES = ['concept', 'concept_ancestor', 'concept_class', 'concept_relationship', 'concept_synonym',
                      'domain', 'drug_strength', 'relationship', 'vocabulary']
 REQUIRED_TABLES = ['person']
-REQUIRED_FILES = [table + '.csv' for table in REQUIRED_TABLES]
+REQUIRED_FILES = map(lambda t: t+'.csv', REQUIRED_TABLES)
```

Comment:
I do not recommend using lambda expressions to replace list comprehensions.  Lambda expressions are slower.  You can open a python interpreter and check this for yourself, with the following:

`import timeit`
`timeit.timeit('[table + ".csv" for table in table_list]', setup='table_list=[]\nfor a in range(0, 2000):\n  table_list.append(str(a))', number=5000)`
`timeit.timeit('map(lambda t: t + ".csv", table_list)', setup='table_list=[]\nfor a in range(0, 2000):\n  table_list.append(str(a))', number=5000)`

Also, changing the name `table` to `t` looses some of the meaning.  A good linting tool will flag this.  If you want to look at pylint install it with `pip install -r dev_requirements.txt` and run `pylint data_steward/common.py` for a list of change suggestions.

Reply:
This change breaks unit tests `test_html_report_person_only`, `test_htm_report_five_person`, and `test_all_files_unparseable_output` in `validation_test.py`.  I think `test_all_files_unparseable_output` fails because of a known bug when the table doesn't exist.

Reply:
I had made that change to keep it consistent with the other constants in the file. Thanks for this @lrwb-aou, it does look like list comprehension is indeed faster and more informative. I'll make the change to list comprehension for all the constants. I'll also fix the unit tests and commit the changes soon.